src/Core/src/Graphics/PaintExtensions.Android.cs

```diff
@@ -82,10 +82,8 @@ internal static bool IsSolid(this AColor color)
 			return color.A is 1;
 		}
 
-		internal static bool IsSolid(this SolidPaint paint)
-		{
-			return paint.Color.Alpha == 1;
-		}
+		internal static bool IsSolid(this SolidPaint? paint) =>
+			paint?.Color.Alpha == 1;
```

Comment:
I fear `Color` is the one being `null` here, because `paint` (named `solidPaint` below) is not null on the caller.

```csharp
var color = solidPaint.Color;
var borderColor = color?.ToPlatform();

_stroke = null;
SetBorderColor(borderColor);
_isBorderSolid = solidPaint.IsSolid();
```

Reply:
`IsSolid(this LinearGradientPaint paint)` and `bool IsSolid(this RadialGradientPaint paint)` will need the same coalesce on the `?.Color` accessor.